src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Microsoft.AspNetCore.Razor.LanguageServer.Test.csproj

```diff
@@ -65,4 +65,9 @@
     <EmbeddedResource Include="Semantic\TestFiles\**\*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Semantic\TestFiles\RangesParamsRazorSemanticTokenInfoServiceTest\" />
+    <Folder Include="Semantic\TestFiles\RangeParamsRazorSemanticTokenInfoServiceTest\" />
+  </ItemGroup>
```

Comment:
Now that you have generated the new baselines, these lines should be able to be removed